provider/resources.go

```diff
@@ -1319,6 +1322,16 @@ func Provider() tfbridge.ProviderInfo {
 			"aws_glue_trigger":                {Tok: awsResource(glueMod, "Trigger")},
 			"aws_glue_workflow":               {Tok: awsResource(glueMod, "Workflow")},
 			"aws_glue_user_defined_function":  {Tok: awsResource(glueMod, "UserDefinedFunction")},
+			"aws_glue_data_catalog_encryption_settings": {
+				Tok: awsResource(glueMod, "DataCatalogEncryptionSettings"),
+				Fields: map[string]*tfbridge.SchemaInfo{
+					"data_catalog_encryption_settings": {
+						CSharpName: "DataCatalogEncryptionSettingsConfig",
+					},
+				},
+			},
+			"aws_glue_ml_transform": {Tok: awsResource(glueMod, "MlTransform")},
```

Comment:
Should it be `MLTransform` (keep 2-char acronym)?

Reply:
agreed!